src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx

```diff
@@ -188,14 +191,17 @@ export const MetadataInstanceEditorCancelChanges: StoryObj<typeof MetadataSideba
     play: async ({ canvasElement }) => {
         const canvas = within(canvasElement);
 
-        const editButtons = await canvas.findAllByRole('button', { name: 'Edit' }, { timeout: 5000 });
+        // Edit buttons contains also template name
+        const editButtons = await canvas.findAllByRole('button', { name: /^edit/i }, { timeout: 5000 });
+        expect(editButtons).toHaveLength(3);
```

Comment:
❓ any reason to using `findAllByRole` here? I see the length check below but most everything else only depends on the first button.  Should the selector be updated to using `Edit My Template` directly?

Reply:
It definitely can, I didn't wanted to change too much in the test. Just fix it.

It started failing after we did fixes for A11y in `metadata-editor` and the `aria-label` for the Edit button got a template name.